conftest.py

```diff
@@ -154,6 +154,8 @@ def setup_regtest_bitcoind(pytestconfig):
         create_wallet = f'{root_cmd} -rpcwait -named createwallet wallet_name={wallet_name} descriptors=false'
     else:
         create_wallet = f'{root_cmd} -rpcwait createwallet {wallet_name}'
+    if bitcoind_version[0] >= 26:
```

Comment:
This is in the wrong place, start_cmd gets called earlier, on line 141

Reply:
So I guess it's 1/ get_bitcoind_version, 2/ check 26 for allowignoredconf, 3/ local_command(start_cmd)

Reply:
Thanks! Moved version check and modifying command above.